### PR TITLE
DTO attached

### PR DESCRIPTION
**Pull request by vandana6878**
DTO have been Created for each entity. Post is working fine.

### CODE REVIEW

File: src/main/java/com/accolite/tfr/controller/TestController.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    public String testCtrl()
+    {
+        return "Working Successfully";
+    }
+
+    @Autowired
+    private ProjectServiceImp projectService;
+    @Autowired
+    private OrganisationServiceImp organisationService;
+
```
Comment (shubham-0) [inline src/main/java/com/accolite/tfr/controller/TestController.java:30]: please remove the commented lines

**Review by shubham-0 — CHANGES_REQUESTED** [review]

File: src/main/java/com/accolite/tfr/controller/TestController.java (hunk 1 above)
Comment (vandana6878) [reply src/main/java/com/accolite/tfr/controller/TestController.java:30]: Changes done as per required. Please check it once.

Comment (shubham-0) [reply src/main/java/com/accolite/tfr/controller/TestController.java:30]: approved

**Review by shubham-0 — APPROVED** [review]
approved

**Pull request merged**